bin/oref0-cron-every-minute.sh

```diff
@@ -60,7 +60,8 @@ find /var/log/openaps/pump-loop.log -mmin +5 | grep pump && (
 ) | tee -a /var/log/openaps/pump-loop.log &
 
 # if the rig doesn't recover after that, reboot:
-oref0-radio-reboot &
+# DEPRECATED WITH 0.7.0. See if we can do without
+# oref0-radio-reboot &
```

Comment:
No, this wasn't deprecated: I re-enabled it for 0.7.0 with commit de462defb4d0c9a8fa6b173cf29febe2e6a165ca

Reply:
i see a lot of radio reboots with the 0.7.0 that seem unnecessary and cause interruption during normal  pump-rig comms.  if it's really needed in some corner cases perhaps move to every 15 minutes? I would like to make it configurable and see if I can do without these reboots.  I think there is still some other issue that makes rebooting without network not boot up the rig without problems.

Reply:
only lengthen the delay will keep a lot of false positives (that hopefully will be cancelled in time). previouse mmweowlink and pump comms could really be messed up after a reboot. I rather see if we can minimize the amount of reboots, perhaps only make them as an option.

How about testing it every 15 minutes and upping the reboot time to 10 minutes. That would give a minimum of 10 minutes and max of 25 minutes in case of pump-rig comms failures. Thoughts?

Reply:
I've never seen any issues caused by a reboot: can you provide more specifics on what problems you've seen as a result of rebooting a 0.7.0 rig?

I'd rather not wait 25 minutes to start looping again if the rig gets into a bad state.  I'm ok with a few extra log messages about scheduled reboots that get canceled before occurring: those don't do any harm, and if someone is looking at their logs closely enough to notice them, they might point to an underlying issue...

Reply:
@scottleibrand 
Issues I now of are:
- older explorer boards have issues with Dexcom G4 USB and the micro usb needs to be inserted after reboot
- edison does not have RTC and during systemstartup internet connection is required in order to set the right time. We should make sure the time from the pump is used in those cases, but that doesn't completely work (yet)

Another reason is that rebooting in case of an issue instead of fixing the root cause is a Microsoft way of doing stuff.

Can we agree on:
- making reboot configurable
- let's try to disable it by default and see if the current 0.7.x codebase has issues that resolve if you reboot the rig
- try to resolve those issues, instead of rebooting
- people who enable the setting should check if the 0.7.0-dev rig reboots in common daily use
- before 0.7.x release decide if we need to use enable reboot by default